pkg/playbook/app/installer/capi.go

```diff
@@ -47,12 +48,20 @@ func NewCAPIInstaller(inst domain.Installation) CAPIInstaller {
 func (r CAPIInstaller) Check(renewBefore string, request domain.PlaybookRequest) (bool, error) {
 	zap.L().Info("checking certificate health", zap.String("format", r.Type.String()), zap.String("location", r.Location))
 
-	friendlyName := request.Subject.CommonName
-	if request.FriendlyName != "" {
-		friendlyName = request.FriendlyName
+	// Get friendly name. If no friendly name is set, get CN from request as friendly name.
+	//  NOTE: This functionality is deprecated, and in a future version will be removed, and CAPIFriendlyName will be req'd
+	friendlyName := r.CAPIFriendlyName
+	if friendlyName == "" {
+		friendlyName = request.Subject.CommonName
```

Comment:
@rvelaVenafi Shouldn't we keep the validation for CommonName being empty, which sets the FriendlyName?

Reply:
Intention was never to validate the CN of the request, rather to get a FriendlyName for the CAPI stored-certificate.
It has been agreed that the value would be mapped to a property of its own: capiFriendlyName.
In order to not keep breaking compatibility, if the attribute is not set, then we take the CN from the request (old logic). A warning has been added at validation-time to make users aware they should explicitly set a FriendlyName